README.md

```diff
@@ -30,22 +30,28 @@ established successfully
 
 ```bash
 % ./tcpgoon --help
-Usage of ./tcpgoon:
+tcpgoon tests concurrent connections towards a server listening on a TCP port
+
+Usage:
+  tcpgoon [flags]
+
+Flags:
   -y, --assume-yes         Force execution without asking for confirmation
   -c, --connections int    Number of connections you want to open (default 100)
-  -d, --debug              Print debugging information to the standard error
-  -t, --dial-timeout int   Connection dialing timeout, in s (default 5)
-  -h, --host string        Host you want to open tcp connections against (Required)
+  -d, --dial-timeout int   Connection dialing timeout, in ms (default 5000)
+  -h, --help               help for tcpgoon
   -i, --interval int       Interval, in seconds, between stats updates (default 1)
-  -p, --port int           Port you want to open tcp connections against (Required)
+  -p, --port int           [Required] Port you want to open tcp connections against
   -s, --sleep int          Time you want to sleep between connections, in ms (default 10)
+  -t, --target string      [Required] Target host you want to open tcp connections against
```

Comment:
do you really like target? For me a host is more concise; target is some kind of "generic", and someone may enter into confusion when specifying the format (host:port as the target value?). I see, in any case, the problem with the abbreviation of host (-h -> -help). Maybe we can just mimic the behaviour of nc or telnet; "destination port" could be the last mandatory argument (without a flag)

Reply:
ack, see this commit 72fd4b1fe4236d4747b38580d837b5b9d43d8383